Translate/En/21-08.md

```diff
@@ -1,118 +1,121 @@
 
-In addition, some of the main factions ……There is little battle capability left for the Japanese, and the underworld people including the integrity knight Renri also are full of creation. Even if it says that there is an advantage of sacred art and sword skill, I do not think it can be repulsed very much.
+In addition, some of the moderate factions…… There is little battle capability left in the Japanese, and the underworld people including the integrity knight Renri also which are all on their last breath(??). Even if there is an advantage from the possession of sacred art and sword skills, I do not think the opposition can be repulsed much.
 
-I can not find any words anymore, so I can only sit in the arm of Solterina.
+I cannot find any words anymore, I can only stand up while gripping Solterina's arm.
 
-With several echoes Loud laughter of the POH has arrived..
+Echoes of loud laughter from PoH arrives...
 
-Beside the lying Kirito, the death god who left a large hole in his chest raised his right hand and a five-finger extended left hand holding the giant friendship kitchen knife at the highest and laughed with his upper body rebelled against him . One day the black clouds are huge swirls in the sky, resources of life which is dissipated in the battlefield are hung down on the tornado and are sucked into the body of POH.
+Beside Kirito, who was lying down, the death god who has a large hole in his chest raised his right hand and all his five-finger extended and his left hand, which is holding the giant friendship kitchen knife at its highest and laughed with his upper body resisting its weight. Huge black clouds swirls in the sky at daytime, a funnel of life resources dissipated in the battlefield forms a tornado as they all are absorbed into PoH's body.
 
-To be precise, it is the right magic sword that absorbs resources. If we can destroy that, the energy supply to the owner will be cut off, the death god who lost the heart should die immediately.
+To be precise, it is the right magic sword that absorbs life resources. If we can destroy that, the energy supply to the owner will be cut off, the death god who lost his heart should die immediately.
 
-However, the situation has already come to the point where even if one of the enemy commanders is defeated it can not be scooped. The mainist factions driven by PoH's inciting words and evil talent, even if the commander leaves, continues fighting until it kills Japanese and underworld people as a source of further anger I will. In the case of
+However, the situation has already come to the point where even if one of the enemy commanders is defeated it can no longer be salvaged. The main factions is driven by PoH's inciting words and evil talent. Even if the commander leaves, they will continue fighting until they kill all the Japanese and Underworld people who are also acting as a source of further anger.
 
-_How should I act? What to do?_
+In the case,
```

Comment:
Some jerk trained Google translate to "translate" isolated space ( ) to a phrase of "In the case of". Funny thing that AI picked it and now I'm seeing an interesting behavior of how it tries to concatenate it to real sentences... anyway, please remove that portion entirely.